deploy/configs/tools/packages.yaml

```diff
@@ -0,0 +1,135 @@
+packages:
```

Comment:
why this is changed in this PR?

Reply:
I don't remember exactly the details, but we had to move the packages.yaml to avoid it being overwritten by one of the steps in the jenkins pipeline.
@matz-e Are we keeping this workaround? Or do you think there is a better solution?

Reply:
@jorblancoa I think I've resolved the CI bug that caused the previous iteration to crash. We should revert back to one `packages.yaml`, and that should be fine (except for rebuilding a whole lot of things)